src/sbmlutils/test/report/test_curated_biomodels.py

```diff
@@ -0,0 +1,2022 @@
+    BIOMD0000000877_SBML,
+    BIOMD0000000878_SBML,
+    BIOMD0000000879_SBML,
+    BIOMD0000000880_SBML,
+    BIOMD0000000881_SBML,
+    BIOMD0000000882_SBML,
+    BIOMD0000000883_SBML,
+    BIOMD0000000884_SBML,
+    BIOMD0000000885_SBML,
+    BIOMD0000000886_SBML,
+    BIOMD0000000887_SBML,
+    BIOMD0000000888_SBML,
+    BIOMD0000000889_SBML,
+    BIOMD0000000890_SBML,
+    BIOMD0000000891_SBML,
+    BIOMD0000000892_SBML,
+    BIOMD0000000893_SBML,
+    BIOMD0000000894_SBML,
+    BIOMD0000000895_SBML,
+    BIOMD0000000896_SBML,
+    BIOMD0000000897_SBML,
+    BIOMD0000000898_SBML,
+    BIOMD0000000899_SBML,
+    BIOMD0000000900_SBML,
+    BIOMD0000000901_SBML,
+    BIOMD0000000902_SBML,
+    BIOMD0000000903_SBML,
+    BIOMD0000000904_SBML,
+    BIOMD0000000905_SBML,
+    BIOMD0000000906_SBML,
+    BIOMD0000000907_SBML,
+    BIOMD0000000908_SBML,
+    BIOMD0000000909_SBML,
+    BIOMD0000000910_SBML,
+    BIOMD0000000911_SBML,
+    BIOMD0000000912_SBML,
+    BIOMD0000000913_SBML,
+    BIOMD0000000914_SBML,
+    BIOMD0000000915_SBML,
+    BIOMD0000000916_SBML,
+    BIOMD0000000917_SBML,
+    BIOMD0000000918_SBML,
+    BIOMD0000000919_SBML,
+    BIOMD0000000920_SBML,
+    BIOMD0000000921_SBML,
+    BIOMD0000000922_SBML,
+    BIOMD0000000924_SBML,
+    BIOMD0000000925_SBML,
+    BIOMD0000000926_SBML,
+    BIOMD0000000927_SBML,
+    BIOMD0000000928_SBML,
+    BIOMD0000000929_SBML,
+    BIOMD0000000930_SBML,
+    BIOMD0000000931_SBML,
+    BIOMD0000000932_SBML,
+    BIOMD0000000933_SBML,
+    BIOMD0000000934_SBML,
+    BIOMD0000000935_SBML,
+    BIOMD0000000936_SBML,
+    BIOMD0000000937_SBML,
+    BIOMD0000000938_SBML,
+    BIOMD0000000939_SBML,
+    BIOMD0000000940_SBML,
+    BIOMD0000000941_SBML,
+    BIOMD0000000942_SBML,
+    BIOMD0000000943_SBML,
+    BIOMD0000000944_SBML,
+    BIOMD0000000945_SBML,
+    BIOMD0000000946_SBML,
+    BIOMD0000000947_SBML,
+    BIOMD0000000948_SBML,
+    BIOMD0000000949_SBML,
+    BIOMD0000000950_SBML,
+    BIOMD0000000951_SBML,
+    BIOMD0000000952_SBML,
+    BIOMD0000000953_SBML,
+    BIOMD0000000954_SBML,
+    BIOMD0000000955_SBML,
+    BIOMD0000000956_SBML,
+    BIOMD0000000957_SBML,
+    BIOMD0000000958_SBML,
+    BIOMD0000000959_SBML,
+    BIOMD0000000960_SBML,
+    BIOMD0000000961_SBML,
+    BIOMD0000000962_SBML,
+    BIOMD0000000963_SBML,
+    BIOMD0000000964_SBML,
+    BIOMD0000000965_SBML,
+    BIOMD0000000966_SBML,
+    BIOMD0000000967_SBML,
+    BIOMD0000000968_SBML,
+    BIOMD0000000969_SBML,
+    BIOMD0000000970_SBML,
+    BIOMD0000000971_SBML,
+    BIOMD0000000972_SBML,
+    BIOMD0000000973_SBML,
+    BIOMD0000000974_SBML,
+    BIOMD0000000975_SBML,
+    BIOMD0000000976_SBML,
+    BIOMD0000000977_SBML,
+    BIOMD0000000978_SBML,
+    BIOMD0000000979_SBML,
+    BIOMD0000000980_SBML,
+    BIOMD0000000981_SBML,
+    BIOMD0000000982_SBML,
+    BIOMD0000000983_SBML,
+    BIOMD0000000984_SBML,
+    BIOMD0000000985_SBML,
+    BIOMD0000000986_SBML,
+    BIOMD0000000987_SBML,
+)
+
+
+sbml_paths = [
```

Comment:
Seriously this is a one liner?
```
sbml_paths = [BIOMODELS_PATH / f"BIOMD0000000{k:0>3}.xml" for k in range(1,988)]
```
Not sure the zero padding is correct, but you can figure this out, i.e. the `{k:0>3}`
Also update to compressed models